src/fake-timers-src.js

```diff
@@ -1259,14 +1259,7 @@ function withGlobal(_global) {
             return uninstall(clock, config);
         };
 
-        clock.methods = config.toFake || [];
-
-        if (clock.methods.length === 0) {
-            // do not fake nextTick by default - GitHub#126
-            clock.methods = keys(timers).filter(function(key) {
-                return key !== "nextTick" && key !== "queueMicrotask";
-            });
-        }
+        clock.methods = config.toFake || keys(timers);
```

Comment:
another technically breaking change I guess - if people passed `toFake: []` before, we faked everything. Seems like a bug fix to me to respect it, but maybe not 😀 If _not_, should just revert the last commit

Reply:
Yeah I think this and the other one should be a major version if we land them.

Reply:
Can open a separate PR for it?

Reply:
> Can open a separate PR for it?

Yes